PostProcessing/Runtime/Effects/MultiScaleVO.cs

```diff
@@ -451,15 +458,17 @@ void PreparePropertySheet(PostProcessRenderContext context)
 
         void CheckAOTexture(PostProcessRenderContext context)
         {
-            if (m_AmbientOnlyAO == null || !m_AmbientOnlyAO.IsCreated() || m_AmbientOnlyAO.width != context.width || m_AmbientOnlyAO.height != context.height)
+            if (m_AmbientOnlyAO == null || !m_AmbientOnlyAO.IsCreated() || m_AmbientOnlyAO.width != context.width || m_AmbientOnlyAO.height != context.height ||
+                m_AmbientOnlyAO.useDynamicScale != context.camera.allowDynamicResolution)
             {
                 RuntimeUtilities.Destroy(m_AmbientOnlyAO);
 
                 m_AmbientOnlyAO = new RenderTexture(context.width, context.height, 0, RenderTextureFormat.R8, RenderTextureReadWrite.Linear)
                 {
                     hideFlags = HideFlags.DontSave,
                     filterMode = FilterMode.Point,
-                    enableRandomWrite = true
+                    enableRandomWrite = true,
+                    useDynamicScale = context.camera.allowDynamicResolution
```

Comment:
Need an ifdef?

Reply:
No, I believe an ifdef is needed here.

Reply:
I believe we do. PPv2 has to stay compatible with 2017.1, and IIRC `allowDynamicResolution` was implemented somewhere around 2017.3?

Reply:
I didn't realize we go back as far as 2017.1 (I'd assumed only 2017.4). I've checked against 2017.1+ versions and updated anywhere that was broken.